huawen-lei/week5/wed/homework-w5d3/app/models/movie.rb

```diff
@@ -0,0 +1,5 @@
+
+class Movie < ApplicationRecord
+    belongs_to :director
```

Comment:
If i delete a director, and the movie remains, there will be an error. consider putting `dependent: :destroy`  or `optional: true` in the directors model